src/data/faq_de.json

```diff
@@ -117,6 +117,70 @@
                     }
                 ]
             },
+            {
+                "title": "Testzertifikat",
+                "id": "test_cert",
+                "accordion": [
+                    {
+                        "title": "Fragen und Antworten zum Testzertifikat",
+                        "anchor": "test_cert_index",
+                        "active": true,
+                        "textblock": [
+                            "<ul><li><a href='#test_cert_why'>Wozu dient das digitales COVID-Testzertifikat?</a></li><li><a href='#test_cert_get'>Wie kann ich ein Testzertifikat anfordern?</a></li><li><a href='#test_cert_rat_and_pcr'>Kann ich Testzertifikate für Schnelltests und PCR-Test anfordern?</a></li><li><a href='#test_cert_when_and_where'>Wann erhalte ich mein Testzertifikat und wo finde ich es?</a></li><li><a href='#test_cert_validity'>Wie lange ist mein Testzertifikat gültig?</a></li><li><a href='#test_cert_privacy'>Wie wird meine Privatsphäre geschützt, wenn ich ein Testzertifikat erstelle?</a></li></ul>"
+                         ]
+                    },
+                    {
+                        "title": "Wozu dient das digitales COVID-Testzertifikat?",
+                        "anchor": "test_cert_why",
+                        "active": true,
+                        "textblock": [
+                            "Ab der Version 2.4 der Corona-Warn-App (CWA) können Sie ein digitales Testzertifikat über die App anfordern. Durch die neue Funktion haben Sie die Möglichkeit, Ihr negatives Testergebnis durch ein offizielles, digitales COVID-Testzertifikat in Form eines QR-Codes in der CWA nachzuweisen. Innerhalb der Länder der Europäischen Union kann es als offiziell gültiger Nachweis eines negativen Testergebnisses dienen, den Sie beispielsweise für Reisen nutzen können oder in anderen gesetzlich vorgeschriebenen Fällen, wie z.B. einem Restaurantbesuch. Sie können dann den QR-Code in der CWA vorzeigen."
+                        ]
+                    },
+                    {
+                        "title": "Wie kann ich ein Testzertifikat anfordern?",
+                        "anchor": "test_cert_get",
+                        "active": true,
+                        "textblock": [
+                            "Das Zertifikat können Sie anfordern, indem Sie einen Test in der Corona-Warn-App (CWA) registrieren: Dazu können Sie auf dem Startbildschirm Ihrer CWA auf 'Test registrieren' tippen und dann auf 'QR-Code scannen'. Nachdem Sie den QR-Code zur Test-Registrierung gescannt haben, öffnet sich automatisch das Fenster zum COVID-Testzertifikat. Dort können Sie das 'Testzertifikat anfordern' oder ablehnen, falls Sie keins erhalten möchten."

```

Comment:
"dem Startbildschirm" => "der Startseite"
to use the name shown in the app under the house icon.

Reply:
ok thanks!